### PR TITLE
June MOMC proposal poll

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

**Comment by LongForWisdom** [comment]
Just go ahead and make any fixes if any are needed.

File: governance/polls/PPG - Open Market Committee Proposal - May 30, 2022.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [MakerDAO Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 30 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+MKR Token Holders may vote in this poll to express support or opposition to the following parameter changes:
```
Comment (TheExistence) [inline governance/polls/PPG - Open Market Committee Proposal - May 30, 2022.md:28]: It's just a proposal. It's very clear how you put it together, but it occurred to me that maybe you could give it this format.

MKR Token Holders may vote in this poll to express support or opposition to the following parameter changes: 

**[Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line)**

Debt Ceiling Decreases

| Vault Type     |   Old |       New | Change |
|----------------|-------|-----------|--------|
| WSTETH-A       | 300 million DAI | **200 million DAI** |  - 100 million DAI|
| DIRECT-AAVEV2-DAI   | 300 million DAI | **100 million DAI** |  - 200 million DAI |
| LINK-A     | 100 million DAI | **50 million DAI** |  - 50 million DAI |

Debt Ceiling Increases

| Vault Type     |   Old |       New | Change |
|----------------|-------|-----------|--------|
| MANA-A    | 10 million DAI | **15 million DAI** |  5 million DAI |


**[Target Borrow Rate](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar)**

Target Borrow Rate Decrease

| Vault Type     |   Old |   New | Change |
|----------------|-------|-------|--------|
| ETH-C          | 3.5%  | **2.75%** | -0.75% |

**[Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap)**

Target Available Debt Increase

| Vault Type     |   Old |   New | Change |
|----------------|-------|-------|--------|
| WSTETH-B         | 15 million DAI | **30 million DAI** | 15 million DAI

**Review by TheExistence — COMMENTED** [review]
Just a few comments :)

File: governance/polls/PPG - Open Market Committee Proposal - May 30, 2022.md (hunk 1 above)
Comment (prose11) [reply governance/polls/PPG - Open Market Committee Proposal - May 30, 2022.md:28]: Like the suggestion, but think with less changes it reads a little better outside of the table. Might be worth figuring out a standard so it's presented the same regardless of the amount of changes tho

**Review by prose11 — APPROVED** [review]
LGTM

**Pull request merged**